salt/states/win_certutil.py

```diff
@@ -65,7 +65,7 @@ def add_store(name, store, saltenv="base"):
         ret["result"] = False
         return ret
 
-    cert_serial = __salt__["certutil.get_cert_serial"](name)
+    cert_serial = __salt__["certutil.get_cert_serial"](name, saltenv)
```

Comment:
```suggestion
    cert_serial = __salt__["certutil.get_cert_serial"](name, saltenv=saltenv)
```

This would also apply to the other calls below